src/CMakeLists.txt

```diff
@@ -356,9 +356,24 @@ if(ENABLE_GUI)
             ${FONTCONFIG_LIBRARIES})
     endif()
 
-    if(MSVC)
-        set_target_properties(solvespace PROPERTIES
-            LINK_FLAGS "/MANIFEST:NO /SAFESEH:NO /INCREMENTAL:NO /OPT:REF")
+    if(WIN32)
+        if(MSVC)
+            set_target_properties(solvespace PROPERTIES
+                LINK_FLAGS "/MANIFEST:NO /SAFESEH:NO /INCREMENTAL:NO /OPT:REF")
+        else(CMAKE_CXX_COMPILER_ID STREQUAL "Clang" AND MSVC_VERSION)
```

Comment:
Ugh, I'm only now noticing that I used `else()` instead of `elseif(). This doesn't break anything, because MINGW and clang-cl simply don't have that flag set, so this would be a no-op. But this should really be changed to `elseif()` at some point in the future, at least for clarity.

Reply:
Done